docs/application/native/guides/internationalization/i18n.md

```diff
@@ -783,17 +783,17 @@ The UCharIter functions return code unit values in the range of 0 - 0xffff. Befo
 
 The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
 
```

Comment:
Line 79: Immutable Index and PluralFormat is supported since Tizen 5.0.

Please change as following:

The Immutable Index and the PluralFormat are supported from Tizen 5.0.

Reply:
Please change "since" to "from"